test/puppetlabs/puppetdb/schema_test.clj

```diff
@@ -44,6 +44,15 @@
        :foo :foo
        :foo (s/required-key :foo)))
 
+(deftest blacklist->vector-test
+  (are [x y] (= x (blacklist->vector y))
+    ["baz" "bar"] "baz, bar"
+    ["foo" "bar"] ["foo" "bar"])
```

Comment:
If you'd rather keep "are", that's fine, but I tend to favor just writing out the "is" statements these days so that the line numbers will be right when they fail.

Reply:
I can rebase and do these with is